src/common/src/config.rs

```diff
@@ -1515,6 +1550,10 @@ pub mod default {
         pub fn max_prefetch_block_number() -> usize {
             16
         }
+
+        pub fn compactor_concurrent_uploading_sst_count() -> Option<usize> {
```

Comment:
Can we set it to a larger value such as 3 or 4?
I think a large task (more than 2GB) holding all handles in memory would cost a lot of memory.
And a small task holding 2 or 3 uploading handle makes no difference.